src/main/java/uk/gov/hmcts/reform/lrdapi/service/impl/LrdServiceImpl.java

```diff
@@ -43,7 +43,8 @@
 
         } else if (StringUtils.isNotBlank(ccdCaseType)) {
 
-            serToCcdCaseType = serviceToCcdCaseTypeAssocRepositry.findByCcdCaseType(ccdCaseType.trim().toUpperCase());
+            serToCcdCaseType = serviceToCcdCaseTypeAssocRepositry
+                .findByCcdCaseTypeIgnoreCase(ccdCaseType.trim().toUpperCase());
```

Comment:
do we need to convert to Upper case as we are ignoring case ? Correct me if I am wrong :)

Reply:
yes, removed upper case

Reply:
thanks!!